pkgs/development/libraries/hunspell/dictionaries.nix

```diff
@@ -967,4 +967,33 @@ rec {
       platforms = platforms.all;
     };
   };
+
+  /* ROMANIAN */
+  ro_RO = ro-ro;
+  ro-ro = mkDict rec {
+    pname = "hunspell-dict-ro-ro";
+    version = "3.3.10";
+    shortName = "ro-ro";
+    dictFileName = "ro_RO";
+    fileName = "${dictFileName}.${version}.zip";
+    shortDescription = "Romanian (Romania)";
+    readmeFile = "README";
+
+    src = fetchurl {
+      url = "https://downloads.sourceforge.net/rospell/${fileName}";
+      sha256 = "7f128d64ea06c9e6711c30b118c0afeefb014d8f33c92daccdf455aba2d04519";
+    };
+
+    nativeBuildInputs = [ unzip ];
+    unpackCmd = ''
+      unzip $src ${dictFileName}.aff ${dictFileName}.dic ${readmeFile} -d ${dictFileName}
+    '';
+
+    meta = with lib; {
```

Comment:
Remove `with lib` it's antipattern. https://github.com/NixOS/nixpkgs/issues/208242
You also use with lib.licenses below so it make it even more redundant.

Reply:
Sorry. I was unsure of the code principles, I just followed a couple of other dictionaries I saw in that file. I'll make my code work without it if is needed.